### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by danforbes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,77 @@
...
+[overview](docs/overview.md) for more details.
 
-It's easy to write blockchains using Substrate, and the overhead of writing parachains' distribution, p2p, database, and synchronization layers is generally high and should be reusable. This project aims to make it easy to write parachains for Polkadot by leveraging the power of Substrate.
+It's easy to write blockchains using Substrate, and the overhead of writing parachains'
+distribution, p2p, database, and synchronization layers should be just as low. This project aims to
+make it easy to write parachains for Polkadot by leveraging the power of Substrate.
 
-Cumulus clouds are shaped sort of like dots and are up in the air, like this project (as it is an initial prototype -- expect a rename when it gets cooler.)
+Cumulus clouds are shaped sort of like dots and are up in the air, like this project (as it is an
+initial prototype -- expect a rename when it gets cooler.)
```
Comment (rphmeier) [inline README.md:13]: I wrote this line in Jan 2019 - I think the name stuck and we can remove this line : ) 

This project doesn't seem "up in the air" anymore

Comment (danforbes) [reply README.md]: Alright I threw a little jazz on it let me know what you think :saxophone:

File: README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ It's easy to write blockchains using Substrate, and the overhead of writing para
 distribution, p2p, database, and synchronization layers should be just as low. This project aims to
 make it easy to write parachains for Polkadot by leveraging the power of Substrate.
 
-Cumulus clouds are shaped sort of like dots and are up in the air, like this project (as it is an
-initial prototype -- expect a rename when it gets cooler.)
+Cumulus clouds are shaped sort of like dots; together they form a system that is intricate,
```
Comment (rphmeier) [inline README.md:12]: poetic : )

Comment (danforbes) [reply README.md:12]: Any changes needed or can I get an approval?

**Pull request merged**